CHANGELOG.md

```diff
@@ -6,12 +6,14 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+The following sections document changes that have been released already:
+
+## 0.1.1 - 2020-03-12
```

Comment:
Actually I think the other year is incorrect too :)

```suggestion
## 0.1.1 - 2021-03-12
```

Reply:
It's becoming a running gag to have wrong release dates ^^

Reply:
Let's just pretend that it's bait to attract helpful friendly contributors :P